src/DiscussionEmbed.jsx

```diff
@@ -75,6 +75,8 @@ export class DiscussionEmbed extends React.Component {
             this.page.title = config.title;
             this.page.remote_auth_s3 = config.remoteAuthS3;
             this.page.api_key = config.apiKey;
+            if (config.language)
+                this.language = config.language;
```

Comment:
The `config.language` prop should also be added to the `DiscussionEmbed.propTypes`.

Reply:
We'll also probably want to add the `language` prop to the `DisqusProps` interface in `types/index.d.ts` to keep the Typescript definitions up-to-date.

Reply:
> The `config.language` prop should also be added to the `DiscussionEmbed.propTypes`.

Since `language` configuration is optional, I suppose I could add it as:
```
language: PropTypes.string
```
without the isRequired flag set right?

I also saw both `remoteAuthS3` & `apiKey` were missing from `DiscussionEmbed.propTypes`, should they also be added?

Reply:
Yes, the `isRequired` flag isn't necessary for the `language` prop.
You are also correct that `remoteAuthS3` & `apiKey` should be included in the  `DiscussionEmbed.proptypes`. You can either include them here or I can add them in a separate PR.